src/libraries/System.Net.Http/src/System.Net.Http.csproj

```
@@ -526,5 +526,4 @@
   <ItemGroup>
     <None Include="Resources\SR.resx" />
   </ItemGroup>
```

```suggestion
  </ItemGroup>

```
Reverting whitespace changes.